ci/lava/tests/test-bsp.py

```diff
@@ -27,7 +27,8 @@ def test_badblocks(self, execute_helper):
             [
                 "shell",
                 'sh -l -c "badblocks -v '
-                "$(/sbin/blkid -L rootfs1 | sed 's/p[0-9]+$//')"
+                "/dev/"
+                "$(cat /config/factory/part-info/MBL_ROOT_DEVICE_NAME_BANK1)"
```

Comment:
Previous version would resolve to "/dev/mmcblk1p3"
This resolves to "/dev/mmcblk1" on zeus-dev build 54.
Is this acceptable?

Reply:
Didn't the `sed 's/p[0-9]+$//` that was previously there strip out the "p3"?

What do we want the test to actually do?

Reply:
I see "INFO: Running badblocks test on /dev/mmcblk1p3" in a test log (https://lava.mbedcloudtesting.com/scheduler/job/70645#results_4327781). Where is that message coming from though? It doesn't look like it's generated by test-bsp.py... Where is it getting the "/dev/mmcblk1p3" from?

Reply:
We discussed this in person. It turns out the `sed 's/p[0-9]+$//` wasn't doing what it looks like, but we've decided to run badblocks on the whole device anyway so we'll leave this as it is.